docs/ui_helpers.rst

```diff
@@ -84,7 +84,7 @@ The template should use `knockout templates <http://knockoutjs.com/documentation
 to render the data you pass back to the view. Each template will have access to
 everything inside of `itemData`. Here's an example:
 
-.. code-block:: html
+.. code-block:: text
```

Comment:
These three changes from code-block:: html to text were a bit disappointing. The first one uses jinja syntax which I assume throws off the html parser. The other two seemed straightforward but in attempting to resolve the warning about syntax highlighting being disabled for these blocks, I couldn't get it to go away. Granted that is a warning we could live with, but at least with `text` it is clear there won't be any syntax highlighting.